pkg/upgrade/upgrade.go

```diff
@@ -400,6 +402,24 @@ func removeCsv(c client.Client, r *rest.Config) error {
 	return nil
 }
 
+func RemoveResource(cfg *rest.Config, applicationNS string) error {
```

Comment:
I think we should keep this `func` generic purpose.

```suggestion
func RemoveResource(cfg *rest.Config, applicationNS string, resource schema.GroupVersionResource, instanceName string) error {
```